Remotion/Web/Core/Res/Themes/NovaViso/HTML/FormGrid.css

```diff
@@ -1,105 +1,95 @@
 /*  Style applied to the form grid table's outer structure  */
-table.formGridTable
+.formGridTable
+{
+  --formgrid-border: var(--remotion-themed-border-width) solid var(--color-standard-separator);
+}
+
+.formGridTable
 {
   border-collapse: collapse;
-  border-width: 0;
   width: 100%;
 }
 
-td.formGridTitleCell,
-td.formGridSubTitleCell
+.formGridTable tr td:not(:last-child, :only-child)
```

Comment:
Using the actual classes is difficult, because if there are one or more columns added (to the left of `formGridLabelsCell`, for example) I cannot possibly tell at this point which classes these `td`s will have. If these cells do have one of the given classes (`formGridLabelsCell`, `formGridMarkersCell`, `formGridControlsCell`), though, then it wouldn't be a problem, of course.

Reply:
I'm not sure I completely get the problem. Maybe it's based on syntax: 

`td.formGridLabelsCell:first-child` does not apply the first TD with the specified css class, but rather to the first TD in the row if it also has the specified css class. So, we should be okay?

![image](https://user-images.githubusercontent.com/6972548/133396172-13228887-768b-4ab4-b5b1-aacda98b4f5b.png)